.github/workflows/tag.yml

```diff
@@ -0,0 +1,93 @@
+name: Tag
+
+on:
+  workflow_dispatch:
+    inputs:
+      branch:
+        description: "The Infinispan branch to checkout when cutting the release."
+        required: true
+        default: "main"
+      version:
+        description: "Release version"
+        required: true
+      nextVersion:
+        description: "Next version"
+        required: true
+      dryRun:
+        description: "Dry run (doesn't push to Central)"
+        type: boolean
+        default: false
+
+jobs:
+  release:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Set up Java
+        uses: actions/setup-java@v4
+        with:
+          java-version: '23'
+          distribution: 'temurin'
+          server-id: ossrh
+          server-username: MAVEN_USERNAME
+          server-password: MAVEN_PASSWORD
+          gpg-private-key: ${{ secrets.INFINISPAN_MAVEN_GPG_ARMORED }}
+          gpg-passphrase: MAVEN_GPG_PASSPHRASE
+
+      - name: Checkout Source
+        uses: actions/checkout@v4
+        with:
+          ref: ${{ github.event.inputs.branch }}
+          ssh-key: ${{ secrets.INFINISPAN_RELEASE_PRIVATE_KEY }}
+
+      - name: Configure Git User
+        run: |
+          git config user.email "infinispan@infinispan.org"
+          git config user.name "Infinispan"
+
+      - name: Update proto.lock files
+        run: |
+          mvn -B clean install -DskipTests -DcommitProtoLockChanges=true -Dcheckstyle.skip
+
+      - name: Set release version
+        run: |
+          mvn -B versions:set -DprocessAllModules=true -DnewVersion=${{ github.event.inputs.version }}
+          mvn -B versions:set-property -Dproperty=version.infinispan -DnewVersion=${{ github.event.inputs.version }}
+          sed -e "s/[0-9]\+\.[0-9]\+\.[0-9]\+\.Final/${{ github.event.inputs.version }}/g" README.md
```

Comment:
`-i` to actually change the file?
also does the regex do what we need?
I noticed that README has badges not aligned: final is 15.0 dev is 15.1

Reply:
I've removed that `sed` line as it wasn't necessary, and fixed the `README.md` file